src/p2f/src/CodeOwls.PowerShell/CodeOwls.PowerShell.Provider/Provider.cs

```diff
@@ -54,6 +56,34 @@ public abstract class Provider : NavigationCmdletProvider,
         IContentCmdletProvider
     {
         private static readonly Dictionary<string, Regex> FilterRegexMap = new Dictionary<string, Regex>(StringComparer.OrdinalIgnoreCase);
+        private static readonly string StringSeparator = (IsWindows) ? "\\" : "/";
+        private static readonly char CharSeparator = (IsWindows) ? '\\' : '/';
```

Comment:
[System.IO.Path.Path ](https://docs.microsoft.com/en-us/dotnet/api/system.io.path?view=netframework-4.7.1) already defines these characters in a cross-platform manner. Please use those definitions instead.

For example, this is defined as Path.DirectorySeparatorChar.

Reply:
Fixed. Thanks Mike.